### PR TITLE
Next release is Fubini

### PR DESCRIPTION
**Pull request by pleroy**

### CODE REVIEW

File: ksp_plugin_adapter/main_window.cs [hunk 1]
```diff
@@ -5,10 +5,10 @@ namespace ksp_plugin_adapter {
 
 internal class MainWindow : SupervisedWindowRenderer {
   // Update this section before each release.
-  private const string next_release_name_ = "Frobenius";
+  private const string next_release_name_ = "Fubini";
   private const int next_release_lunation_number_ = 250;
```
Comment (eggrobin) [inline ksp_plugin_adapter/main_window.cs]: Increment.

File: ksp_plugin_adapter/main_window.cs [hunk 2]
```diff
@@ -272,18 +272,6 @@ internal class MainWindow : SupervisedWindowRenderer {
   }
 
   private void RenderLoggingSettings() {
-#if FROBENIOID
-    string trace = "";
-    if (FlightGlobals.ActiveVessel &&
-        adapter_.PluginRunning() &&
-        plugin.HasVessel(FlightGlobals.ActiveVessel.id.ToString())) {
-      trace =
-          plugin.VesselGetPileUpTrace(FlightGlobals.ActiveVessel.id.ToString());
```
Comment (eggrobin) [inline ksp_plugin_adapter/main_window.cs]: Remove (all the way to the `PileUp`).

File: ksp_plugin_adapter/main_window.cs (hunk 1 above)
Comment (pleroy) [reply ksp_plugin_adapter/main_window.cs]: Done.

File: ksp_plugin_adapter/main_window.cs (hunk 2 above)
Comment (pleroy) [reply ksp_plugin_adapter/main_window.cs]: Done.

**Pull request merged**